src/test/scala/samples/APIUnitTests.scala

```diff
@@ -0,0 +1,72 @@
+package samples
+
+import org.junit.runner.RunWith
+import org.specs2.mutable._
+import org.specs2.runner._
+import java.util.concurrent.ConcurrentHashMap
+
+import api.MyMap
+import api._
```

Comment:
Are both these import needed? IntelliJ would warn you about import optimisation by greying out unnecessary imports and unused values/methods. You'll also see a yellow box in the top right corner having a list of these warnings. Try to make sure all files have a green tick at the top right.